### ISSUE TITLE
Iterations in Multiple Team Forecaster spreadsheet?

### ISSUE THREAD

**Issue by mike-rogers** [FocusedObjective/FocusedObjective.Resources#29]
First let me say that I absolutely love the spreadsheet. :smile:

Is there a reason that the reporting data focuses on 'weeks' instead of 'iterations'? There seem to be no date-based calculations (although granted I'm not an Excel Master and I probably failed to look deeply enough) and it's a cognitive strain to remember "where you see 'week' instead think 'iteration'".

Would it be possible to replace 'week' with 'iteration' throughout?

**Comment by FocusedObjective** [comment]
Good idea. It should be in the unit chosen for the delivery pace estimates or data. Or at least configurable in the settings worksheet.

Thanks,
Troy
